tasks/create-advisory/create-advisory.yaml

```diff
@@ -71,6 +71,20 @@ spec:
             exit 1
         fi
 
+        # Ensure RHSA is only used if CVEs are provided
+        NUM_CVES=$(jq '.content.images[]?.cves.fixed // 0 | length' <<< "$advisoryData" \
+          | awk '{sum=sum+$0} END{print sum}')
```

Comment:
You can do the sum directly with `jq`. Also, while it works with the default `0`, you're expecting an array, so defaulting to `[]` seems more correct.
```suggestion
        NUM_CVES=$(jq '[.content.images[]?.cves.fixed // [] | length] | add' <<< "$advisoryData")
```
But if you prefer awk, that's fine with me.

Reply:
Maybe I am doing something wrong?
```
$ cat mapping.json | jq '[.mapping.components[]?.tags // [] | length | add'
jq: error: syntax error, unexpected end of file (Unix shell quoting issues?) at <top-level>, line 1:
[.mapping.components[]?.tags // [] | length | add                                              
jq: 1 compile error
$ cat mapping.json | jq '.mapping.components[]?.tags // 0 | length' | awk '{sum=sum+$0} END{print sum}'
5
```
Happy to remove the `awk` if the jq add works though

Reply:
Much better, thanks for the suggestion